webapp/channels/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -68,6 +71,7 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
     sidebarRightWidthHolder: React.RefObject<HTMLDivElement>;
     previous: Partial<Props> | undefined = undefined;
     focusSearchBar?: () => void;
+    private previousActiveElement: HTMLElement | null = null;
```

Comment:
I don't think it's likely a big deal, but I did want to mention that this could hold onto a reference to a DOM node after it gets detached from the DOM. Given we only hold onto a single node at once, I doubt this would cause us to leak memory, but I wanted to mention it in case it does come up later. If that ever does cause an issue, we could probably fix that by using a WeakRef anyway

Reply:
noted 🖊️